test-case/check-pause-resume-suspend-resume.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+##
+## Case Name: check-pause-resume-suspend-resume
+## Preconditions:
+##    N/A
+## Description:
+##    test playback with pause/resume as well as suspend/resume
+##    fake pause/resume with expect on aplay process
+##    when aplay is paused, have system enter suspend state for 5 secs
+##    resume from suspend state
+##    release aplay from paused state, then pause again
+##    repeat
+## Case step:
+##    1. aplay process is started
+##    2. aplay process is then paused via mock spacebar press via expect
+##    3. once aplay is paused, have system enter suspend state for 5 secs
+##    4. resume system from suspend state and unpause aplay process
+##    5. repeat from #2 - #4 50x
+## Expect result:
+##    no errors occur for either process
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['p']='pcm'          OPT_DESC_lst['p']='playback pcm. Example: hw:0,0'
```

Comment:
change this into a complex param? like
`-d Phw:0,0` for playback and `-d Chw:0,0` 
or
`-P hw:0,0` for playback `-C hw:0,0` for capture

@Bin-QA to keep this test simple, let try to test with one pipeline. We can use some wrapper to wrap with tplg

Reply:
sounds good.

Reply:
Done.